config/roles.php

```diff
@@ -14,18 +14,20 @@
 
 $adminAssignable = [
     Role::ARTIST,
-    Role::DEVELOPER,
     Role::DEVELOPER_JUNIOR,
     Role::DEVELOPER_STAFF,
     Role::DEVELOPER_VETERAN,
+    Role::DEVELOPER,
```

Comment:
I think these were meant to be sorted alphabetically.

Reply:
Thanks, fixed in latest.